packages/vite/migrations.json

```diff
@@ -29,6 +29,12 @@
       "description": "Changes the testFile config in the vite:test exectutor from a string to an array of strings",
       "cli": "nx",
       "implementation": "./src/migrations/update-16-4-1-update-test-file-config/update-16-4-1-test-file-config"
+    },
+    "16-5-0-change-ts-paths-plugin": {
+      "version": "16.5.0-beta.0",
```

Comment:
Increment this version, to beta.4 or something? Because if you're already on `next`, which is `beta.2` then it will not get it I think.